### PR TITLE
Tarun samanta/43

### PR DESCRIPTION
**Pull request by tarunsamanta2k20**
Requires https://github.com/pticostaricags/FairPlaySocial/issues/42 to be implemented

Create a new table named GroupMember, with the following columns

GroupId
MemberApplicationUserId
IOriginatorInfo interface columns

#43

### CODE REVIEW

File: src/FairPlaySocialSln/FairPlaySocial.Database/dbo/Tables/GroupMember.sql [hunk 1]
```diff
@@ -0,0 +1,9 @@
+CREATE TABLE [dbo].[Group]
```
Comment (efonsecab) [inline src/FairPlaySocialSln/FairPlaySocial.Database/dbo/Tables/GroupMember.sql:1]: Please fix the table name, set it to GroupMember

File: src/FairPlaySocialSln/FairPlaySocial.Database/dbo/Tables/GroupMember.sql [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+(
+    [GroupId] BIGINT NOT NULL CONSTRAINT PK_Group_M PRIMARY KEY IDENTITY,
+    [MemberApplicationUserId]  BIGINT NOT NULL,
```
Comment (efonsecab) [inline src/FairPlaySocialSln/FairPlaySocial.Database/dbo/Tables/GroupMember.sql:4]: Set a Foreign Key to [ApplicationUser].[ApplicationUserId]

Also, please create a unique index for the combination of columns
* GroupId
* MemberApplicationUserId

Comment (tarunsamanta2k20) [reply src/FairPlaySocialSln/FairPlaySocial.Database/dbo/Tables/GroupMember.sql:4]: Hey,
I have made required changes. Please let me know if any other modification required.

**Comment by tarunsamanta2k20** [comment]
Done

**Review by efonsecab — APPROVED** [review]
@tarunsamanta2k20 Approved, thanks for your contribution.

**Pull request merged**